### PR TITLE
Lab1 angular v14 upgrades

### PR DESCRIPTION
**Pull request by suhussai**
*Issue #, if available:*

*Description of changes:*
Lab1 angular v14 upgrades

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Lab1/client/Application/README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
-# Application
+# Dashboard
```
Comment (ujwalbukka) [inline Lab1/client/Application/README.md:2]: Should be "Application"

File: Lab1/client/Application/angular.json [hunk 2]
```diff
@@ -1,165 +1,112 @@
...
               "budgets": [
+                {
+                  "type": "initial",
+                  "maximumWarning": "2mb",
+                  "maximumError": "2mb"
+                },
                 {
                   "type": "anyComponentStyle",
-                  "maximumWarning": "6kb"
+                  "maximumWarning": "2kb",
```
Comment (ujwalbukka) [inline Lab1/client/Application/angular.json:50]: keep 6kb

File: Lab1/client/Application/angular.json [hunk 3]
```diff
@@ -1,165 +1,112 @@
...
-          }
+          },
+          "defaultConfiguration": "development"
         },
         "extract-i18n": {
           "builder": "@angular-devkit/build-angular:extract-i18n",
           "options": {
-            "browserTarget": "ng:build"
+            "browserTarget": "dashboard:build"
           }
```
Comment (ujwalbukka) [inline Lab1/client/Application/angular.json:89]: should be "application:build" right?

File: Lab1/scripts/deployment.sh [hunk 4]
```diff
@@ -3,111 +3,109 @@
...
+    ex -sc '%s/s3_bucket = .*/s3_bucket = \"'$SAM_S3_BUCKET'\"/|x' samconfig.toml
   fi
 
   echo "Validating server code using pylint"
-  python3 -m pylint -E -d E0401 $(find . -iname "*.py" -not -path "./.aws-sam/*")
+  python3 -m pylint -E -d E0401 "$(find . -iname "*.py" -not -path "./.aws-sam/*")"
   if [[ $? -ne 0 ]]; then
-    echo "****ERROR: Please fix above code errors and then rerun script!!****"
-    exit 1
+    echo "****WARNING: Please fix code errors found in above files.****"
```
Comment (ujwalbukka) [inline Lab1/scripts/deployment.sh:61]: If validation fails echo error and exit.

File: Solution/Lab1/client/Application/karma.conf.js [hunk 5]
```diff
@@ -25,7 +25,7 @@ module.exports = function (config) {
       suppressAll: true // removes the duplicated traces
     },
     coverageReporter: {
-      dir: require('path').join(__dirname, './coverage/Application'),
+      dir: require('path').join(__dirname, './coverage/dashboard'),
```
Comment (ujwalbukka) [inline Solution/Lab1/client/Application/karma.conf.js:28]: should be "Application" right

File: Solution/Lab1/client/Application/package.json [hunk 6]
```diff
@@ -1,73 +1,46 @@
 {
-  "name": "application",
+  "name": "dashboard",
```
Comment (ujwalbukka) [inline Solution/Lab1/client/Application/package.json:2]: "application"

File: Solution/Lab1/client/Application/src/index.html [hunk 7]
```diff
@@ -1,13 +1,29 @@
...
+  <head>
+    <!-- https://github.com/aws/aws-amplify/issues/678 fix: -->
+    <script>
+      if (global === undefined) {
+        var global = window;
+      }
+    </script>
+    <!-- https://github.com/aws/aws-amplify/issues/678 fix end-->
+    <meta charset="utf-8" />
+    <title>Dashboard</title>
```
Comment (ujwalbukka) [inline Solution/Lab1/client/Application/src/index.html:12]: should be "Application" right?

**Review by ujwalbukka — APPROVED** [review]
Added comments around few more typo's. Please fix them

File: Lab1/client/Application/src/app/app.component.ts [hunk 8]
```diff
@@ -1,37 +1,21 @@
...
+    private domSanitizer: DomSanitizer
+  ) {
+    this.matIconRegistry.addSvgIcon(
+      'saas-commerce',
+      this.domSanitizer.bypassSecurityTrustResourceUrl('./assets/logo.svg')
+    );
   }
-
+  title = 'dashboard';
 }
```
Comment (ujwalbukka) [inline Lab1/client/Application/src/app/app.component.ts:21]: should be 'application' right?

**Comment by suhussai** [comment]
Replaced by https://github.com/aws-samples/aws-serverless-saas-workshop/pull/22

**Pull request closed without merging**